packages/world/src/systems/CraftSystem.sol

```diff
@@ -29,7 +29,6 @@ contract CraftSystem is System {
     require(recipe.inputTypes.length > 0, "Recipe not found");
 
     if (!recipe.stationTypeId.isNull()) {
-      require(station.exists(), "This recipe requires a station");
```

Comment:
Why dont we need to check this? is it because checking the object type is enough?

Reply:
yep